contrib/pylightning/tests/test_version.py

```diff
@@ -0,0 +1,7 @@
+import lightning
+
+
+def test_version():
```

Comment:
We can actually ensure this programmatically: `setup.py` can run arbitrary code to detect stuff, so we could just import the `lightning` module and use `__version__` in the call to `setup()`

Reply:
OK, great!  Wasn't sure if that would work if the dependencies for lightning module aren't installed yet, but maybe that's not what setup.py is for.